src/Symfony/Component/ExpressionLanguage/SerializedParsedExpression.php

```diff
@@ -34,6 +34,10 @@ public function __construct($expression, $nodes)
 
     public function getNodes()
     {
+        $options = null;
+        if (PHP_VERSION_ID >= 70000) {
+            $options = array('allowed_classes' => false);
+        }
         return unserialize($this->nodes);
```

Comment:
I removed that bit for now because `$this-nodes` contains Node-classes. I will have to check whether I can restrict allowed_classes here.